spec/lib/coach/request_serializer_spec.rb

```diff
@@ -34,5 +35,39 @@
         expect(sanitized).to eq('value')
       end
     end
+
+    context "as an instance" do
+      before do
+        @mock_request = instance_double("Request", :request,
+          uuid: nil, method: nil,
+          format: nil, remote_ip: nil,
+          filtered_parameters: nil,
+          filtered_env: { "foo" => "bar", "HTTP_bar" => "bar" }
+        )
+      end
+
+      it "can be initialized with one argument" do
+        expect(described_class.new('instance')).to(
+          be_an_instance_of(Coach::RequestSerializer))
+      end
+
+      describe "the #serialize method" do
```

Comment:
This looks great, but I think we use `describe "#serialize" do` as the convention in other spec files.

Would also be nice to make `described_class.new(@mock_request).serialize` the subject within this describe block, and to move `allow(@mock_request).to receive(:fullpath) { raise }` into a `before` block (and possibly use RSpec's `and_raise` method).